src/components/SummaryBox/SummaryBoxHeading/SummaryBoxHeading.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface SummaryBoxHeadingProps {
+  heading: string
```

Comment:
instead of a heading prop, can we use `children` here? That way the api can look like `<SummaryBoxHeading>My Heading Content</SummaryBoxHeading>`

That'll also allow more flexibility over just `string` in case someone wanted to otherwise stylize or pass in custom tags.

Reply:
@Igarfinkle Did you decide to use `heading` prop anyway instead of `children`? I liked @brandonlenz 's suggestion

Reply:
either way, a comment here to explain why it was resolved without changes would be helpful to future viewers.